lib/rspec/support/object_inspector.rb

```diff
@@ -55,6 +74,25 @@ def self.register(klass, &block)
       def self.registered_klasses
         @registered_klasses ||= {}
       end
+      class ArrayInspector
```

Comment:
Not a merge blocker, but a blank line before this class def would be nice.  Having it smashed together with the prior thing makes it harder to read.